ethereum-explorer/Makefile

```
@@ -0,0 +1,11 @@
+.PHONY: build
+build:
+	cargo build --target wasm32-unknown-unknown --release
+
+.PHONY: protogen
+protogen:
+	substreams protogen ./substreams.yaml --exclude-paths="sf/substreams,google"
+
+.PHONY: package
+package: build
```

Let's normalize to be the `<command>` part of `substreams <command>` so `pack` here.
```suggestion
.PHONY: pack
pack: build
```